code/src/App.js

```diff
@@ -1,9 +1,80 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Summary } from "./Summary";
+import './app.css'
+
 
 export const App = () => {
+  const [happy, setHappy] = useState("");
+  const [proud, setProud] = useState("");
+  const [smile, setSmile] = useState("");
+  const [showSummary, setShowSummary] = useState(false);
+
+  const smileOptions = ["Yes", "No", "Maybe"]
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    setShowSummary(true);
+  };
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <main>
+      {!showSummary ? (
+        <form onSubmit={handleSubmit}>
+          <header id="header">
+            <h3>Survey</h3>
+            <h1>Reflections of your day</h1>
+            <a href="#happy"><img className="arrow" src="../assets/arrow.png" alt="arrow"></img></a>
+          </header>
+          <section className="section-content happy" id="happy">
+            <h2>How was your day?</h2>
+            <select
+              onChange={event => setHappy(event.target.value)}
+              value={happy}
+              required
+            >
+              <option value="">Select answer</option>
+              <option value="an ok day">Ok</option>
+              <option value="a great day">Great</option>
+              <option value="the best day ever">The best</option>
+            </select>
+            <button><a href="#proud">Next</a></button>
```

Comment:
You don't need a button wrapping a link :)
```suggestion
<a href="#proud">Next</a>
```